userguide/install.rst

```diff
@@ -999,115 +999,91 @@ option. You can permanently boot from disk by removing the
+ESXi server. Enter the username and password created when installing
+ESXi to log into the interface. After logging in, go to *Storage* to
+upload the %brand% :file:`.iso`.
+Click :guilabel:`Datastore browser` and select a datastore for the
+%brand% :file:`.iso`. Click :guilabel:`Upload`. Use the
+pop-up browser to upload the %brand% :file:`.iso` from the host system.
 
-Click :guilabel:`Next` and enter a name for the virtual machine. Click
-:guilabel:`Next` and highlight a datastore. An example is shown in
-:numref:`Figure %s <esxi_datastore_fig>`.
-Click :guilabel:`Next`. In the screen shown in
-:numref:`Figure %s <esxi_os_fig>`,
-click :guilabel:`Other`, then select a FreeBSD 64-bit architecture.
 
+Click :guilabel:`Create / Register VM` to create a new VM. The *New
+virtual machine* wizard opens:
 
-.. _esxi_datastore_fig:
+#. **Select creation type**: Select
+   :literal:`Create a new virtual machine` and click :guilabel:`Next`.
 
-.. figure:: images/vmware-storage.png
+   .. _esxi creation type:
 
-   Select Datastore
+   .. figure:: images/esxi_create_type.png
 
+#. **Select a name and guest OS**: Enter a name for the VM. Leave ESXi
+   compatibility version at the default. Select :literal:`Other` as the
+   Guest OS family. Select
+   :literal:`FreeBSD12 or later versions (64-bit)` as the Guest OS
+   version. Click :guilabel:`Next`.
 
-.. _esxi_os_fig:
+   .. _exsi name and guest OS:
 
-.. figure:: images/vmware-operating-system.png
+   .. figure:: images/esxi_name_os.png
 
-   Select Operating System
+#. **Select storage**: Select a datastore for the VM. The datastore
+   must be at least 32 GiB.
 
+   .. _esxi select storage:
 
-Click :guilabel:`Next` and create a virtual disk file of **8 GiB** to
-hold the %brand% operating system, as shown in
-:numref:`Figure %s <esxi_create_disk_fig>`.
+   .. figure:: images/esxi_select_storage.png
 
+#. **Customize settings**: Enter an amount of memory and virtual
+   storage. Ensure they meet the recommended *8 GiB* of RAM and
+   *32 GiB* of storage. Select :literal:`Datastore ISO file` from
+   the :guilabel:`CD/DVD Drive 1` drop-down. Use the Datastore browser
+   to select the uploaded %brand% :file:`.iso`. Click
+   :guilabel:`Next`.
 
-.. _esxi_create_disk_fig:
+   .. _esxi customize settings:
 
-.. figure:: images/vmware-create-os-disk.png
+   .. figure:: images/esxi_custom_settings.png
 
-   Create Disk for the Operating System
+#. **Ready to complete**: Review the VM settings. Click
+   :guilabel:`Finish` to create the new VM.
 
+   .. _esxi ready to complete:
 
-Click :guilabel:`Next` and :guilabel:`Finish`. The new virtual machine
-is listed in the left frame. Right-click the virtual machine and
-select :guilabel:`Edit Settings` to access the screen shown in
-:numref:`Figure %s <esxi_vm_settings_fig>`.
+   .. figure:: images/esxi_ready_complete.png
 
+Additional disks can be added to a VM. Right-click the VM and click
```

Comment:
"Additional" and "added" are redundant.  This can just be a "To add more disks to a VM, ..."